src/components/molecules/TriggerDailyRunButton.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import { useInterval } from "usehooks-ts";
+import { handleResult } from "../../util";
+import fetch from "node-fetch";
+import Button from "../atoms/buttons/Button";
+import styled from "styled-components";
+
+export const ONE_MINUTE_IN_MS = 60000;
+
+const TriggerButton = styled(Button)`
+  text-align: center;
+`;
+
+const TriggerDailyRunButton = () => {
+  const [buttonEnabled, setButtonEnabled] = useState(false);
+  const [buttonNotice, setButtonNotice] = useState<undefined | string>();
+  const [interval, setInterval] = useState(0);
+
+  useInterval(() => {
+    setInterval(ONE_MINUTE_IN_MS);
+    fetch("/api/sharkster/check-breakouts-state")
+      .then(handleResult)
+      .then(({ isIdle, message = null }) => {
+        setButtonEnabled(isIdle);
```

Comment:
Is a future implementation to only trigger one daily run per day, and then disable this button if already performed? Or should it be possible to perform multiple daily runs in one day?

Reply:
It should be possible to execute this multiple times on the same day due to changes of the configuration at Sharkster.